Sources/XCRemoteCache/Dependencies/DependencyProcessor.swift

```diff
@@ -72,7 +72,7 @@ class DependencyProcessorImpl: DependencyProcessor {
 
     private func classify(_ files: [URL]) -> [Dependency] {
         return files.map { file -> Dependency in
-            let filePath = file.path
+            let filePath = file.resolvingSymlinksInPath().path
```

Comment:
maybe indeed we should switch to `BUILT_PRODUCTS_DIR`. For now we can rely on a symlink.